lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -32,6 +32,9 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         self.assertTrue(course_tools)
         self.assertEquals(course_tools[0]['analytics_id'], 'edx.bookmarks')
 
+        dates_widget = response.data.get('dates_widget')
+        self.assertTrue(dates_widget)
```

Comment:
I think this should actually check some of the data being returned inside of the dates_widget, rather than just the existence of the dates widget. It could be full of chaos right now and we would never know. See something like https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_home_api/dates/v1/tests/test_views.py#L39 for an example of testing data inside.